modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/messaging/IndividualSegmentsChecker.java

```diff
@@ -215,7 +215,7 @@ private void _checkIndividualSegmentMemberships(
 
 				individuals.forEach(
 					individual -> {
-						Long userId = _getUserIdOptional(
+						Long userId = _getUserId(
```

Comment:
@W2jx There's no need to keep a separate rename commit, just rename it directly when you change it to not return `Optional`.